avocado/utils/network/interfaces.py

```diff
@@ -781,3 +782,35 @@ def validate_ipv4_netmask_format(self, netmask):
                 return False
             first_bit = False
         return True
+
+    def validate_macaddr(self, mac_str):
```

Comment:
please change it to validate_mac_addr just for readability., rest all looks good